src/Tag/StorageInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace FOS\HttpCache\Tag;
+
+/**
+ * Implementors of this interface associate cache entry identifiers
+ * with tags, retrieve cache entry identifiers for tags and remove tags.
+ */
+interface StorageInterface
+{
+    /**
+     * Associate a list of tags with the given cache identifier.
+     * The expriry represents the TIL for the cache entry, it might
+     * be the value of the S-MAXAGE header for example.
+     *
+     * The identifier can be any scalar value which can be associated with a
+     * unique HTTP cache entry.
+     *
+     * @param string[] $tags
+     * @param mixed $identifier
+     * @param integer $expiry
+     * @return void
+     */
+    public function tagCacheId(array $tags, $identifier, $expiry = null);
```

Comment:
this is the same method signature as ManagerInterface has. what are the differences?